### ISSUE TITLE
CR_KEYWORD entries -- various issues

### ISSUE THREAD

**Issue by pmusser** [mitodl/ocw_oer_export#122]
### Expected Behavior
Row 745 (https://ocw.mit.edu/courses/3-014-materials-laboratory-fall-2006 -- "Materials Laboratory") should have keywords listed. Other entries have no keywords listed.

### Current Behavior
the entry for row 745 returns `<same as F05 Version>` rather than any keywords

### Steps to Reproduce
1. See most recent export

### Possible Solution
Populate with the following keywords: `materials science|quantum mechanics|thermodynamics` for 745.

**Comment by pdpinch** [comment]
I think the solution is to edit this line to put in the correct keywords: 

https://github.com/mitodl/ocw_oer_export/blob/7dcf74e0657229e32d921315b80271f1eaa46683/ocw_oer_export/mapping_files/fm_keywords_export.csv#L226

**Comment by ibrahimjaved12** [comment]
@pdpinch Yes, that is correct.

However, there are 4 more courses that are lacking keywords.

1. Social Theory and Analysis (https://ocw.mit.edu/courses/21a-750j-social-theory-and-analysis-fall-2011/) -- looks like an unpublished course
2. Sorting Truth From Fiction: Civic Online Reasoning (https://ocw.mit.edu/courses/res-cms-504-sorting-truth-from-fiction-civic-online-reasoning-spring-2021/)
3. Competency-Based Education: The Why, What, and How (https://ocw.mit.edu/courses/res-cms-502-competency-based-education-the-why-what-and-how-spring-2020/)
4. IIT Joint Entrance Exam Preparation (https://ocw.mit.edu/courses/res-tll-009-iit-joint-entrance-exam-preparation-fall-2022)

I think the first course needs to be removed from search index? It's the 2nd course in search with a 404 link: https://ocw.mit.edu/search/?q=Social+Theory+and+Analysis
Could you please confirm.

For complete CSV data on these 4 courses:
https://docs.google.com/spreadsheets/d/1K93ScLMFFAxkoV4aORLv_dRcI9Kx4AYFP1Q3X1YQKXA/edit?gid=2088832682#gid=2088832682

**Comment by pdpinch** [comment]
You're right about #1. I think we can just remove it from https://api.learn.mit.edu/admin/learning_resources/learningresource/4079/ by unchecking "Published" but I'd like to confirm with the Learn team.  

For #2, #3 and #4 we simply don't have keywords for them. It was around this time that we stopped creating keywords for courses because Google ignored them for search. 

We don't even have a keyword field for courses in the current schema. 

If you can fix https://github.com/mitodl/ocw_oer_export/issues/122#issuecomment-2486400534 I think we'll have done what we can and we can close this issue.

**Comment by pmusser** [comment]
Just a note on compatibility re: keywords, the CR_Keywords field in OER
Commons is a required field, but that can be satisfied with something
general that would apply to all of them since it doesn't use a
controlled vocabulary. E.g. if it simplifies things, appending "MIT Open
Learning" to the field would solve that requirement (and also help convey
the rebrand).

That doesn't necessarily help with the CR_Subject, but I thought it might
not hurt to mention. Thanks again for tackling these!
